atlasdb-api/src/test/java/com/palantir/atlasdb/metrics/MetricPublicationArbiterTest.java

```diff
@@ -55,6 +55,7 @@ public void metricsWithOneFilterAreFiltered() {
     }
 
     @Test
+    @SuppressWarnings("DistinctVarargsChecker")
```

Comment:
`DistinctVarargsChecker` throws even for methods, despite that they may return a unique object